recipe/build.sh

```diff
@@ -1,37 +1,44 @@
+#!/bin/sh
 CFLAGS=$(echo "${CFLAGS}" | sed "s/-march=[a-zA-Z0-9]*//g")
 CFLAGS=$(echo "${CFLAGS}" | sed "s/-mtune=[a-zA-Z0-9]*//g")
 
 # Avoid sorting LDFLAGS
 sed -i.bak 's/LDFLAGS := $(sort $(LDFLAGS))//g' common.mk
 
-case $target_platform in
-    osx-*)
-        export CC=$BUILD_PREFIX/bin/clang
-        ./configure --prefix=$PREFIX --enable-cblas --enable-threading=pthreads intel64
-        make CC_VENDOR=clang -j${CPU_COUNT}
-        make install
-        make check -j${CPU_COUNT}
-        ;;
-    linux-*)
-        ln -s `which $CC` $BUILD_PREFIX/bin/gcc
-        export CC=$BUILD_PREFIX/bin/gcc
-        ./configure --prefix=$PREFIX --enable-cblas --enable-threading=pthreads x86_64
-        make CC_VENDOR=gcc -j${CPU_COUNT}
-        make install
-        make check -j${CPU_COUNT}
-        ;;
-    win-*)
-        export LIBPTHREAD=
-        ./configure --disable-shared --enable-static --prefix=$PREFIX --enable-cblas --enable-threading=pthreads --enable-arg-max-hack x86_64
-        make -j${CPU_COUNT}
-        make install
-        make check -j${CPU_COUNT}
- 
-        ./configure --enable-shared --disable-static --prefix=$PREFIX --enable-cblas --enable-threading=pthreads --enable-arg-max-hack x86_64
-        make -j${CPU_COUNT}
-        make install
-        mv $PREFIX/lib/libblis.lib $PREFIX/lib/blis.lib
-        mv $PREFIX/lib/libblis.a $PREFIX/lib/libblis.lib
-        mv $PREFIX/lib/libblis.*.dll $PREFIX/bin/
-        ;;
-esac
+if [[ "$CC" == *"-gnu-"* ]]
+then
+  CC_VENDOR=gcc
+else
+  CC_VENDOR=clang
+  # Response files don't seem to work with windows so we have to activate the arg max hack
+  ARG_MAX_HACK="--enable-arg-max-hack"
+  unset CFLAGS
+  unset LDFLAGS
+  unset CXXFLAGS
```

Comment:
Remove these

Reply:
@isuruf: It's really still a draft and very very much a work in progress (because the failing platforms are not platforms I can access easily unfortunately, so no local troubleshoot).

So I would really recommend that you wait until I remove the `draft` tag to get something that is not a moving target (cluttered with *ad hoc* `echo`s, `cat`s and `unset`s that definitely won't be there then).